### PR TITLE
add optional CI test fo darwin

### PR DESCRIPTION
**Pull request by baude**
run gofmt and make for darwin to prevent regressions from new code for a potential
darwin build.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by baude** [comment]
ok this is ready for review

File: .papr.yml [hunk 1]
```diff
@@ -47,3 +47,39 @@ tests:
...
+    - git
+    - glib2-devel
+    - gnupg
+    - golang
+    - libassuan-devel
+    - make
+    - ostree-devel
+    - skopeo-containers
+
+required: false
```
Comment (rhatdan) [inline .papr.yml]: Why is this optional, Don't we want to block PRS that break this?

Comment (baude) [reply .papr.yml]: I was thinking optional initially ... if others want it required, easy change to make here or later.

**Comment by rhatdan** [comment]
Well I still think it should be required, but maybe we wait until we get buildah 1.2 out.
@rh-atomic-bot

**Pull request closed without merging**